samples/snippets/src/main/java/com/example/firestore/snippets/QueryDataSnippets.java

```diff
@@ -644,4 +644,9 @@ Query filterNotIn() {
     // [END fs_filter_not_in]
     return query;
   }
+
+  /** Closes the gRPC channels associated with this instance and frees up their resources. */
+  void close() throws Exception {
```

Comment:
Does this need to be overridden? It seems like the method in `BaseIntegrationTest` does the right thing already.

Reply:
Yes,because ```QueryDataSnippets```  class also have instance variable with name db of type ```Firestore``` means ```BaseIntegrationTest``` have different instance while ```QueryDataSnippets``` class have a different instance of ```Firestore```.